.github/workflows/release.yml

```diff
@@ -1,7 +1,7 @@
 name: Publish package on NPM
 on:
   release:
-    types: [published]
+    types: [released]
```

Comment:
This is updated because the `published` event is trigger for both release and pre-release.
Using `released` instead will ensure we only target releases and not pre-releases.